compiler/circle-quantizer/src/CircleQuantizer.cpp

```diff
@@ -180,9 +181,16 @@ int entry(int argc, char **argv)
 
   if (arser.get<bool>("--verbose"))
   {
+#if defined(__MINGW32__)
```

Comment:
Actually we have never used compiler directives in our compiler frontend.
And I hope not to use them afterwards. 
Reason: These make code readability hard and also development hard.

Reply:
I understand your points :-) I'll make a draft for this with a detailed explaination.

Reply:
Ah, before the draft, let me explain shortly.

from https://sourceforge.net/p/mingw/mailman/message/35495004/

1) MSVCRT.DLL exports putenv(), (strictly _putenv(), but MinGW defines
   an import library alias with the POSIX.1-2008-XSI compatible name),
   which is semantically similar to the Unix System V function of the
   same name.  However, there is no setenv(), nor any semantically
   similar API, provided by any Windows DLL.

2) Neither putenv(), nor setenv() is required by ISO-C, so, even
   though both are specified by POSIX, it isn't considered imperative
   for MinGW to provide what isn't already provided by MSVCRT.DLL.

The points are
- setenv() is not implemented in MinGW because MinGW supports ISO-C spec.
- putenv() can be used instead of setenv() (actually putenv() <- _putenv())

While building on windows, I've faced these problems.

/cc @mhs4670go and @seanshpark

Reply:
Before the draft, we need to discuss
1. how much do we contribute to building on windows. I personally work on windows so I'm doing this task. But, ONE officially doesn't support Windows. How much do we contribute to windows? (of course, I can contribute this as a personal developer NOT an official ONE developer.)
2. Anyway, if I do this task, how to do it? @llFreetimell suggests making a new directory like `WinUtil` and wrapping functions like `setenv()`.

Reply:
I think would be better to post a separate issue for this --> #8308

Reply:
Besides supporting for Windows, I don't think I will accept codes using like `#if defined(__MINGW32__)`. As I wrote, introducing this one will eventually permit codes like this and that will lead hell. (which I had experienced long time ago that I don't want to experience again)

You can consider supporting multi platform as, provide PAL (platform adaptation layer), detect/select platfrom from CMake.